tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -951,3 +952,82 @@ def test_post_email_notification_when_data_is_empty_returns_400(
         assert error_msg == 'phone_number is a required property'
     else:
         assert error_msg == 'email_address is a required property'
+
+
+def test_post_notifications_saves_email_to_queue(client, notify_db_session, mocker):
+    save_email_task = mocker.patch("app.celery.tasks.save_api_email.apply_async")
+
+    service = create_service(service_id='539d63a1-701d-400d-ab11-f3ee2319d4d4', service_name='high volume service')
+    template = create_template(service=service, content='((message))', template_type=EMAIL_TYPE)
+    data = {
+        "email_address": "joe.citizen@example.com",
+        "template_id": template.id,
+        "personalisation": {"message": "Dear citizen, have a nice day"}
+    }
+    response = client.post(
+        path='/v2/notifications/email',
+        data=json.dumps(data),
+        headers=[('Content-Type', 'application/json'), create_authorization_header(service_id=service.id)]
+    )
+
+    json_resp = response.get_json()
+
+    assert response.status_code == 201
+    assert json_resp['id']
+    assert json_resp['content']['body'] == "Dear citizen, have a nice day"
+    assert json_resp['template']['id'] == str(template.id)
+    save_email_task.assert_called_once_with([mock.ANY], queue='save-api-email-tasks')
```

Comment:
Optional: Should we also add an assertion to show it's not saving it to the DB as well as adding it to the queue?

Reply:
It hasn't been saved to the db yet, it's been put on a queue to be saved to the db

Reply:
Sorry, to be clear, assert that it HASN'T been saved to the DB. As in we aren't trying to add it to both the DB and also the queue at the same time?